App.tsx

```diff
@@ -5,6 +5,10 @@ import { NavigationContainer } from "@react-navigation/native";
 import useCachedResources from "./src/hooks/useCachedResources";
 import MainStackNavigator from "./src/navigation/MainStackNavigator";
 
+import * as SplashScreen from "expo-splash-screen";
+
+SplashScreen.preventAutoHideAsync();
+
```

Comment:
Good work!
This may not be necessary as Expo will automatically load the splash screen when needed. 
You can remove it.